server/src/promo-dialog.js

```diff
@@ -0,0 +1,56 @@
+const React = require("react");
+const PropTypes = require("prop-types");
+const { Localized } = require("fluent-react/compat");
+const sendEvent = require("./browser-send-event.js");
+
+exports.PromoDialog = class PromoDialog extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    if (this.props.display) {
+      return <div id="promo-dialog-panel" className="promo-panel" >
+        <div className={`${this.props.promoType} promo-box default-color-scheme`}>
+        <a className="box-close" aria-label="Close Notification" onClick={this.closePanel.bind(this)}></a>
+        <Localized id="promoTitle">
+          <span className="title">Take Note!</span>
+        </Localized>
+        <Localized id="promoMessage">
+          <p className="message">
+            Updated editing tools let you crop, highlight, and even add text to your shot.
+          </p>
+        </Localized>
+        <span className="promo-star"></span>
+         <Localized id="promoLink">
+          <a id="promo-link" aria-label="Try Edit Tool" className="message link" onClick={this.closePanel.bind(this)}>
+            Give it a try
+          </a>
+        </Localized>
+        <span className="promo-star"></span>
```

Comment:
Instead of two spans and a link, just make this a regular paragraph with emoji at the start and end. There's no need for the fancy `::before` CSS stuff.

`<p>✨ <Localized id="promoLink">Give it a try</Localized> ✨</p>`